NDB.Covid19/NDB.Covid19.Droid/Views/Settings/SettingsGeneralActivity.cs

```diff
@@ -32,6 +32,7 @@ class SettingsGeneralActivity : AppCompatActivity
         protected override void OnCreate(Bundle savedInstanceState)
         {
             base.OnCreate(savedInstanceState);
+            this.Title = SettingsGeneralViewModel.SETTINGS_GENERAL_TITLE;
```

Comment:
What's this one?

Reply:
TalkBack announces the page title on navigation. This page was missing this title, so it was inconsistent with the other ones